docs/development_environment.md

```diff
@@ -12,8 +12,8 @@ The easiest way to get started with development is to use Visual Studio Code wit
 
 - [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
 - Docker
-  -  For Linux, macOS, or Windows 10 Pro/Enterprise/Education use the [current release version of Docker](https://docs.docker.com/install/)
-  -   Windows 10 Home requires [WSL 2](https://docs.microsoft.com/windows/wsl/install-win10#update-to-wsl-2) and the current Edge version of Docker Desktop (see instructions [here](https://docs.docker.com/docker-for-windows/wsl-tech-preview/)). This can also be used for Windows Pro/Enterprise/Education.
+  -  For Linux, macOS, or Windows 10/11 Pro/Enterprise/Education use the [current release version of Docker](https://docs.docker.com/install/)
```

Comment:
```suggestion
  -  For Linux, macOS, or Windows (Pro/Enterprise/Education) use the [current release version of Docker](https://docs.docker.com/install/)
```
We should just drop the number

Reply:
I'm not sure about that because it won't work on windows 7,8 or lower. I guess we can also write this as "10 or higher"

Reply:
Anything lower than 10 is now EOL, we should not consider those

Reply:
I agree with dropping the number especially as docker requires Windows 10 as a minimum version  (2004 for everything except Enterprise/Education where 1909 can be used). I also think we shouldn't explicitly reference Windows 11 until Docker lists it as supported